velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -227,22 +227,22 @@ TEST_F(NthValueTest, invalidFrames) {
   auto vectors = makeRowVector({
       makeFlatVector<int32_t>(size, [](auto /* row */) { return 1; }),
       makeFlatVector<int32_t>(size, [](auto row) { return row % 50; }),
-      makeFlatVector<int64_t>(size, [](auto row) { return row % 5; }),
+      makeFlatVector<int64_t>(size, [](auto row) { return -9; }),
```

Comment:
You will need to comment out the row parameter like on line 228 to avoid a linter error.

Reply:
Thanks for the suggestion, updated here. In line 84, the addition of 1 to keep the frame offset positive is only reverted and the param row is still used.